src/command-testing.js

```diff
@@ -60,6 +87,9 @@ window.addEventListener('keyup', (event) => {
             // TODO: need to modify this behavior to be completely different
             // if the popup box isn't shown yet..
             if (!multipleOccurs) {
+                // send a message to the code to grab the most recent tabs
+                var imageUrls = chrome.runtime.sendMessage(null, {grabImages: true}).then().then();
```

Comment:
Not sure what the then().then() is doing. Can we just await it as a promise instead?